ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -87,35 +106,48 @@ def extract_mid_slice_and_convert_coordinates_to_heatmaps(path, suffix, aim=-1):
             heatmap = imed_maths.heatmap_generation(label_array[:, :], 10)
             arr_pred_ref_space = imed_loader_utils.reorient_image(np.expand_dims(heatmap[:, :], axis=0), 2, lab, nib_ref_can)
             nib_pred = nib.Nifti1Image(arr_pred_ref_space, lab.affine)
-            nib.save(nib_pred, os.path.join(path, 'derivatives', 'labels', t[i], 'anat', t[i] + suffix +
-                                            '_mid_heatmap' + str(aim) + '.nii.gz'))
+            nib_pred = rotate_nifti(nib_pred)
+            nib.save(nib_pred, os.path.join(path, 'derivatives', 'labels', t[i], 'anat', t[i]+'_rec-mid' +suffix+ '_heatmap'+str(aim)
+                                            + '.nii.gz'))
         else:
             pass
 
 
 def get_parser():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-p", "--path", dest="path", required=True, type=str,
-                        help="Path to bids folder",
-                        metavar=imed_utils.Metavar.file)
-    parser.add_argument("-s", "--suffix", dest="suffix", required=True, type=str,
-                        help="""Suffix of the input file as in
-                                sub-xxxSUFFIX.nii.gz (E.g., _T2w)""",
-                        metavar=imed_utils.Metavar.str)
-    parser.add_argument("-a", "--aim", dest="aim", default=-1, type=int,
-                        help="""-1 or positive int. If set to any positive int,
-                                only label with this value will be taken into account""",
-                        metavar=imed_utils.Metavar.int)
```

Comment:
These changes to the `add_argument` statements seem unnecessary:

* `metavar` was removed
* The `required` status has changed for `-p` and `-s`
* The default value has changed from `-1` to `0` for `-a`. 
* Some lines were reformatted.

Please revert these changes unless they are absolutely necessary, so that your PR will stay focused and small.

Reply:
In the future, please only change what is absolutely necessary, and try to avoid making any small formatting/style tweaks unless it is relevant for your PR.

Reply:
Done.